src/loggly.tracker.js

```diff
@@ -27,8 +27,15 @@
     }
     
     function setTag(tracker, tag){		
-        tracker.tag = tag;		
-    }	
+        tracker.tag = tag;
+        if (tracker.mutableTags) {
```

Comment:
It seems this won't work if they initialize the library with mutableTags=false and tag=some value. We want to allow it to set during initialization.